web_src/css/editor-markdown.css

```diff
@@ -16,6 +16,9 @@
   user-select: none;
   padding: 5px;
   cursor: pointer;
+  border: none;
+  background: none;
+  color: var(--color-text);
```

Comment:
Ah yes, the conflict with #23896 you mentioned.

Reply:
Will remove this when https://github.com/go-gitea/gitea/pull/23896 lands hopefully before this one.

Reply:
Done, color will stay as the native `<button>` elements need it to prevent browser UA style to color them.